src/main/kotlin/com/personio/synthetics/builder/SyntheticTestBuilder.kt

```diff
@@ -103,6 +103,15 @@ abstract class SyntheticTestBuilder(
         parameters.locations = locations.map { it.value }
     }
 
+    /**
+     * Sets the locations for the synthetic test
+     * @param locationItems List of locations
+     */
+    @Deprecated("Please use `publicLocations` method", ReplaceWith("publicLocations(*locationItems)"))
```

Comment:
Better to use the named arguments. 
` @Deprecated(message = "The `<old function> is deprecated in favor of new `publicLocations`. Please use `publicLocations` function",
        replaceWith = ReplaceWith(expression = "publicLocations(*locationItems)"))`